From 35bcac7b39e8ee2c3360c9409537219831c299f9 Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Tue, 31 Oct 2023 11:16:04 -0700
Subject: [PATCH] Correct ncurses.h path

---
 src/watch.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/watch.c b/src/watch.c
index 5c159a9..8bebc86 100644
--- a/src/watch.c
+++ b/src/watch.c
@@ -53,7 +53,7 @@
 # define _XOPEN_SOURCE_EXTENDED 1
 # include <wchar.h>
 # include <wctype.h>
-# include <ncursesw/ncurses.h>
+# include <ncurses.h>
 #else
 # include <ncurses.h>
 #endif	/* WITH_WATCH8BIT */
-- 
2.42.0

